### PR TITLE
added string keypath documetnation to immutability helpers that can u…

### PR DESCRIPTION
**Pull request by ipbrennan90**
**What kind of change does this PR introduce?**
- Documentation

**Does this PR introduce a breaking change?**
- No

Just adding to the documentation for the immutability helpers to make it clear that get, set, and update can all take arrays of key strings or a string keypath.

### CODE REVIEW

File: docs/api/immutability-helpers.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 1. [Overview](#overview)
 2. [API](#API)
 
-## Overview
+# Overview
```
Comment (nhunzaker) [inline docs/api/immutability-helpers.md]: Could you keep this as an H2?

File: docs/api/immutability-helpers.md [hunk 2]
```diff
@@ -58,10 +58,13 @@ let state = {
 let planets = get(state, 'planets')
 console.log(planets) // { earth, mars }
 
-// Or an array of keys to access a deeply nested value
+// an array of keys to access a deeply nested value
```
Comment (nhunzaker) [inline docs/api/immutability-helpers.md]: Could you capitalize the a here?

Comment (nhunzaker) [inline docs/api/immutability-helpers.md]: Hmmm. This is inconsistent. What if we just capitalized the first character of every comment in the file?

Comment (ipbrennan90) [inline docs/api/immutability-helpers.md]: sounds good

**Pull request merged**